packages/runtime/container-runtime/src/summary/documentSchema.ts

```diff
@@ -298,7 +313,7 @@ function checkRuntimeCompatibility(
 }
 
 function and(
-	currentDocSchema: IDocumentSchemaCurrent,
+	currentDocSchema: IDocumentSchema,
```

Comment:
Let's nename this to not overload the meaning of the word "current".  How about `persistedDocSchema`